src/main/java/marquez/service/models/JobType.java

```diff
@@ -0,0 +1,7 @@
+package marquez.service.models;
+
+public enum JobType {
```

Comment:
I would define the job type with the `Job` class along for `Job.Type.BATCH`, `Job.Type.STREAM`, and `Job.Type. SERVICE`.